java/daikon/tools/jtb/AnnotateVisitor.java

```diff
@@ -342,27 +348,29 @@ public void visit(FieldDeclaration n) {
     //                     requires_invs, ensures_invs);
     // }
 
-    return new /*@Nullable*/ InvariantsAndModifiedVars[] {requires_invs, ensures_invs};
+    return new /*@Nullable*/ InvariantsAndModifiedVars[] {requires_invs, ensures_invs, throw_invs};
   }
 
   public void insertAlso(Node n) {
     addComment(n, "@ also" + lineSep, true);
   }
 
   // n must be a MethodDeclaration or ConstructorDeclaration
-  public void insertBehavior(Node n) {
+  public void insertBehavior(Node n, boolean exceptional) {
     class InsertBehaviorVisitor extends DepthFirstVisitor {
       Node n;
       boolean behaviorInserted;
+      boolean exceptionalBehavior = false;
 
-      public InsertBehaviorVisitor(Node n) {
+      public InsertBehaviorVisitor(Node n, boolean exceptB) {
         super();
+        this.exceptionalBehavior = exceptB;
```

Comment:
Use the same variable name.  That reduces cognitive load and makes the code easier to read.